.yarnrc

```diff
@@ -1,3 +1,3 @@
 disturl "https://electronjs.org/headers"
-target "11.4.1"
+target "11.4.2"
```

Comment:
@deepak1556 I bumped the electron version for testing but this should likely be part of a separate PR